internal/services/subscription/parse/location.go

```diff
@@ -0,0 +1,72 @@
+package parse
```

Comment:
we can use the Subscription ID as the resource ID for this, so this isn't needed?

Reply:
Sure. Updated to use the Subscription ID as the resource ID and removed this file